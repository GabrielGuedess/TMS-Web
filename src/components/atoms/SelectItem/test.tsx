import { render } from '@testing-library/react';
import * as Select from '@radix-ui/react-select';

import { SelectItem } from '.';

describe('<SelectItem />', () => {
  it('should render the select item correctly', () => {
    const { container } = render(
      <Select.Root open>
        <Select.Content>
          <SelectItem value="value">value</SelectItem>
        </Select.Content>
      </Select.Root>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
