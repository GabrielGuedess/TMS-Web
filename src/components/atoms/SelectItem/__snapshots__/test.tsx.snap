// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SelectItem /> should render the select item correctly 1`] = `
<div
  style="display: flex; flex-direction: column; position: fixed;"
>
  <div
    data-state="open"
    dir="ltr"
    id="radix-:r0:"
    role="listbox"
    style="box-sizing: border-box; max-height: 100%; display: flex; flex-direction: column; outline: none; pointer-events: auto;"
    tabindex="-1"
  >
    <div
      aria-labelledby="radix-:r1:"
      aria-selected="false"
      class="flex cursor-pointer items-center gap-2 rounded-md px-3 py-2 text-sm text-comet-500 outline-none hover:bg-primary-400/[.08] hover:text-primary-400 focus:bg-primary-400/[.08] focus:text-primary-400 dark:text-dark-300"
      data-radix-collection-item=""
      data-state="unchecked"
      role="option"
      tabindex="-1"
    >
      <span
        id="radix-:r1:"
      >
        value
      </span>
    </div>
  </div>
</div>
`;
